tools/azure-pipelines/build-apache-repo.yml

```diff
@@ -69,7 +69,7 @@ stages:
             name: Default
           e2e_pool_definition:
             vmImage: 'ubuntu-20.04'
-          environment: PROFILE="-Dflink.hadoop.version=2.10.2"
+          environment: PROFILE="-Dflink.hadoop.version=2.10.2 -Penable-adaptive-scheduler"
```

Comment:
Why do we want to activate the adaptive-scheduler profile here?

Reply:
That's a debug commit which I added to the PR to have a CI run with the AdaptiveScheduler enabled. I'm not planning to merge this commit.

The goal is to have a CI run that actually contains the changes of this PR. That way, we can verify that the tests are actually enabled now (even before merging the PR).

Reply:
Oh wow I completely missed the debug commit 🤦 - Thanks :)